docker-compose.yaml

```diff
@@ -11,9 +11,6 @@ services:
       - 3030:3000
     volumes:
       - ./:/oh
-      - /oh/node_modules
```

Comment:
Here I want node_modules to not be overwritten by mounting the local one, because we already installed in the build step.

Reply:
Better solutions are appreciated!